Installation/cmake/modules/CGAL_CreateSingleSourceCGALProgram.cmake

```diff
@@ -69,11 +69,6 @@ function(create_single_source_cgal_program firstfile )
     add_to_cached_list( CGAL_EXECUTABLE_TARGETS ${exe_name} )
 
     target_link_libraries(${exe_name} PRIVATE CGAL::CGAL)
-    foreach(comp ${CGAL_REQUESTED_COMPONENTS})
-      if(TARGET CGAL::CGAL_${comp})
-        target_link_libraries(${exe_name} PRIVATE CGAL::CGAL_${comp})
-      endif()
-    endforeach()
```

Comment:
That patch will break everything that relies on `create_single_source_cgal_program` to link with `CGAL_Core`, or `CGAL_ImageIO`.

Reply:
then it will be pointed out in the CI and I will be able to add the missing calls to `target_link_libraries()`, if it is actually needed.

Reply:
Actually no: a .cpp may be surrounded by tests of `CGAL_USE_CORE`. We need another way to detect which .cpp use CGAL_Core, or other libraries.